deployment/aws-batch/docker-compose.yml

```diff
@@ -0,0 +1,12 @@
+version: '2'
+services:
+  update-job-defs:
```

Comment:
Can we add this service to `docker-compose.yml` so that all services are defined there?